.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request]"
```

Comment:
```suggestion
title: "[Feature Request] Title"
```

Reply:
I was under the impression that this text is just appended to the front of the text that people enter.  I wonder if your suggestion would give "[Feature Request] Title Please add some new feature"?

Reply:
Sorry,  think I misunderstood :-).  Done.